test/integration/test_upgrade.py

```diff
@@ -59,8 +59,7 @@ def testUpgrade(self):
         self.upgrade(*args)
 
     @pytest.mark.slowtest
-    @pytest.mark.skipif(True,
-        reason='OMERO not supported on Python 3.6')
+    @pytest.mark.skipif(True, reason='OMERO not supported on Python 3.6')
```

Comment:
Does this still hold if you've bumped to Py38?

Reply:
I have not re-activated the tests in this PR. I have actually skipped a good number due to URL. 
This will be a second body of work

Reply:
> I have not re-activated the tests in this PR

Fair enough. The comment might just be slightly confusing.